src/main/java/appeng/crafting/CraftingLinkNexus.java

```diff
@@ -27,16 +27,16 @@
 public class CraftingLinkNexus
 {
 
-	public final String CraftID;
-	boolean canceled = false;
-	boolean done = false;
-	int tickOfDeath = 0;
-	CraftingLink req;
-	CraftingLink cpu;
+	private final String craftID;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "craftID" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)